composer.json

```diff
@@ -6,7 +6,10 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "1.12.*|1.13.*"
+        "sylius/sylius": "1.14.*",
+        "league/flysystem-bundle": "^2.4",
```

Comment:
Why we need those two libraries?

Aren't they needed just for tests, therefore `require-dev` only?

edit: Hmm, I see that `league/flysystem-bundle` is required by `sylius/sylius` itself, so we do not need to require it explicitly, because we do not depend on it in source code, just enabling it in bundles.

Same way the `winzou/state-machine-bundle` is already core dependency of Sylius itself and that is enough.

Remove those additional requirements from composer.json